BDArmory/Control/BDGenericAIBase.cs

```diff
@@ -292,7 +292,7 @@ protected void GetGuardTarget()
                 UpdateWeaponManager();
             if (weaponManager != null && weaponManager.vessel == vessel)
             {
-                if (weaponManager.guardMode && weaponManager.currentTarget != null)
+                if (weaponManager.guardMode && (weaponManager.currentTarget != null && !weaponManager.currentTarget.isMissile))
```

Comment:
Won't this make missiles untargetable by point defence turrets?

Reply:
No. Turrets don't care what the pilotAI targetVessel is, only the WM guardTarget. Adding the !isMissile check here means that the AI won't cancel the attack command and begin engagement if the missile is the only detected enemy vessel

Reply:
OK